packages/devextreme-cli/src/templates/vue-v2/application/devextreme.json

```diff
@@ -1,5 +1,6 @@
 {
   "applicationEngine": "vue",
+  "version": "2",
```

Comment:
I do not like this format. 'version' is the first-level option now but affected only vue template now. If we propose a template-based option here, we should separate them. 
e.g. `{ "applicationEngine": "vue", "vue": { "version": 2 }}`

Reply:
maybe `"option": { "version": 2 }`?

Reply:
`option` is the common. My idea `devextreme[devextreme["applicationEngine"]].version`. 'version' in this case is specific for `devextreme["applicationEngine"]`